src/panels/lovelace/hui-root.ts

```diff
@@ -364,8 +364,8 @@ class HUIRoot extends LitElement {
           display: flex;
         }
         #view > * {
-          flex: 1;
-          width: 100%;
+          flex: 1 1 100%;
+          max-width: 100%;
```

Comment:
How does the page look with 1 column? Since the background is on this element, not having it be 100% will mess with the width ?

Reply:
base width is still 100%, it would be the same as:
```
          flex: 1;
          width: 100%;
          max-width: 100%;
```

Reply:
but the `flex-grow: 1` would also make it 100% width, so it should not be necessary at all

Reply:
I see.

Reply:
CSS hell. Sometimes it's easier to just remove all rules and built it up again. Oh well. 

We should start adding comments to CSS rules like this to make sure we know why they are there.